client/src/context/user/UserState.js

```diff
@@ -0,0 +1,77 @@
+import React, { useReducer } from 'react'
+
+import UserContext from './UserContext'
+import UserReducer from './UserReducer'
+import { GET_SELECTED_USER, CLEAR_SELECTED_USER } from '../types'
+
+import axios from 'axios'
+
+const UserState = (props) => {
+  const initialState = {
+    selectedUser: {
+      id: '',
+      firstName: '',
+      lastName: '',
+      birthday: { month: '', day: '', year: '' },
+      age: '',
+      bio: '',
+      occupation: '',
+    },
+  }
+
+  const [state, dispatch] = useReducer(UserReducer, initialState)
+
+  // Get Selected user
+  const getSelectedUser = async (selectedUserId) => {
```

Comment:
does this method need to be here? can it go into an actions file/folder for the UserState. Also  something that has come to mind. It looks kind of messy here since it doesn't really depend on this component to be able to function i think